packages/components/src/components/Table/TableHeaderCell.tsx

```diff
@@ -0,0 +1,16 @@
+import { spacingsPx, typography } from '@trezor/theme';
+import styled, { css } from 'styled-components';
+
+export const TableHeaderCell = styled.div<{ $alignRight?: boolean }>`
```

Comment:
This and TableRowCell are very similar. I'd make them one component and change the few details with a prop (a variant perhaps?)

Reply:
I was think about the same thing that I can pass props to it but I like it more when it's different in name so it's more explicit.

Reply:
I understand, but they are almost the same:)

Reply:
If you want to, you can make a TableCell with a prop and then the two other components which would use the TableCell component. Just to make it more DRY.